ipatests/test_integration/test_installation.py

```diff
@@ -357,6 +358,53 @@ def test_ipa_ca_crt_permissions(self):
         assert owner == "root"
         assert group == "root"
 
+    def test_is_ipa_configured(self):
```

Comment:
There is the FileBackup which could be used as a context manager in Pytest fixture or directly.
I mean if some of the assertions fail then `SYSRESTORE_STATEFILE` ends up in an unpredictable state.

Reply:
It could but since the installation falls back to the old method it should be able to cleanup properly. If it doesn't then that's a good thing and demonstrates a bug.